rpcs3/xinput_pad_handler.cpp

```diff
@@ -415,28 +415,31 @@ void xinput_pad_handler::ThreadProc()
 
 			// The left motor is the low-frequency rumble motor. The right motor is the high-frequency rumble motor.
 			// The two motors are not the same, and they create different vibration effects. Values range between 0 to 65535.
-			int idx_l = profile->switch_vibration_motors ? 1 : 0;
-			int idx_s = profile->switch_vibration_motors ? 0 : 1;
+			size_t idx_l = profile->switch_vibration_motors ? 1 : 0;
+			size_t idx_s = profile->switch_vibration_motors ? 0 : 1;
 
-			int speed_large = profile->enable_vibration_motor_large ? pad->m_vibrateMotors[idx_l].m_value * 257 : vibration_min;
-			int speed_small = profile->enable_vibration_motor_small ? pad->m_vibrateMotors[idx_s].m_value * 257 : vibration_min;
+			u16 speed_large = profile->enable_vibration_motor_large ? pad->m_vibrateMotors[idx_l].m_value : static_cast<u16>(vibration_min);
+			u16 speed_small = profile->enable_vibration_motor_small ? pad->m_vibrateMotors[idx_s].m_value : static_cast<u16>(vibration_min);
 
-			m_dev->newVibrateData = m_dev->newVibrateData || m_dev->largeVibrate != speed_large || m_dev->smallVibrate != speed_small;
+			m_dev->newVibrateData |= m_dev->largeVibrate != speed_large || m_dev->smallVibrate != speed_small;
 
 			m_dev->largeVibrate = speed_large;
 			m_dev->smallVibrate = speed_small;
 
 			// XBox One Controller can't handle faster vibration updates than ~10ms. Elite is even worse. So I'll use 20ms to be on the safe side. No lag was noticable.
-			if (m_dev->newVibrateData && (clock() - m_dev->last_vibration > 20))
+			using namespace std::chrono_literals;
+			using namespace std::chrono;
+
+			if (m_dev->newVibrateData && (duration_cast<milliseconds>(high_resolution_clock::now() - m_dev->last_vibration) > 20ms))
```

Comment:
this detaches the comment from the actual code.
Also why use 2 namespaces for just two calls?

Reply:
I can remove `using namespace std::chrono`, but I honestly don't know how to use `20ms` literal without including the namespace. I can move the comment directly above the condition.

Reply:
did you try Timer.h

Reply:
you can use .count() I guess

Reply:
It's not about not knowing what to substitute it with, I admit I like the way `20ms` looks. When looks are not a concern, `std::chrono::milliseconds(20)` produce equivalent code, as per:
https://en.cppreference.com/w/cpp/chrono/operator%22%22ms

Reply:
You could use `std::chrono::milliseconds(20)`, perhaps storing it as a variable so that the line doesn't get too long. But using the namespace is fine too imho.